tests/spec/errors.spec.js

```diff
@@ -135,15 +137,15 @@ describe('Filer.Errors', function() {
   });
 
   it('should include all expected properties by default', function() {
-    var err = new Filer.Errors.ENOENT();
+    let err = new Filer.Errors.ENOENT();
```

Comment:
This could actually be `const` vs. `let`, since `err` will never change during the function scope.